### PR TITLE
rework home page

### PR DESCRIPTION
**Pull request by lilyanB**
image
image

### CODE REVIEW

File: web/massastation/.env.dev [hunk 1]
```diff
@@ -1,3 +1,3 @@
 VITE_BASE_APP=""
-VITE_BASE_API=""
+VITE_BASE_API="https://station.massa/"
```
Comment (mazmassa) [inline web/massastation/.env.dev:2]: @Thykof is that right!?

File: web/massastation/src/pages/Index/Index.tsx [hunk 2]
```diff
@@ -1,20 +1,126 @@
...
+        ]}
+        components={[
+          <PluginWallet
+            key="wallet"
+            isActive={pluginWalletIsInstalled}
+            title="Massawallet"
+            iconActive={<WalletActive />}
+            iconInactive={<WalletInactive />}
+            onClickActive={() =>
+              navigate('/plugin/massalabs/wallet/web-app/index')
```
Comment (mazmassa) [inline web/massastation/src/pages/Index/Index.tsx:118]: Not your fault but I guess you missed the `routeFor` wrapping. We need that for production links.

```suggestion
              navigate(routeFor('/plugin/massalabs/wallet/web-app/index'))
```

don't forget to import. It's under /utils

`import { routeFor } from '../../../utils';`

File: web/massastation/src/pages/Index/Index.tsx [hunk 3]
```diff
@@ -1,20 +1,126 @@
...
+        Decentralization made easy
+      </div>
+      <div className="flex space-x-8 pb-10">
+        <Button
+          preIcon={<FiGlobe />}
+          customClass="w-96"
+          onClick={() => navigate('/web/search')}
+        >
+          <div className="flex items-center mas-buttons">
+            Search Massa ecosystem
```
Comment (mazmassa) [inline web/massastation/src/pages/Index/Index.tsx:80]: Can we use the I18n!?

You can define the keys here: `web/massastation/src/i18n/en_US.json`

then import:

`import Intl from '../../i18n/i18n';`

then use it:

`{Intl.t('pagename.function.key')}`

Let me know if you need help with that

File: web/massastation/.env.dev (hunk 1 above)
Comment (Thykof) [reply web/massastation/.env.dev:2]: I think it's no necessary

File: web/massastation/src/pages/Index/Index.tsx (hunk 2 above)
Comment (Thykof) [reply web/massastation/src/pages/Index/Index.tsx:118]: maybe not here, let's try

File: web/massastation/src/pages/Index/Index.tsx (hunk 3 above)
Comment (lilyanB) [reply web/massastation/src/pages/Index/Index.tsx:80]: oh yes, I see what you mean! Nice!
It's done

File: web/massastation/src/pages/Index/Index.tsx (hunk 2 above)
Comment (lilyanB) [reply web/massastation/src/pages/Index/Index.tsx:118]: done

File: web/massastation/.env.dev (hunk 1 above)
Comment (lilyanB) [reply web/massastation/.env.dev]: done

File: web/massastation/src/pages/Index/Index.tsx (hunk 2 above)
Comment (lilyanB) [reply web/massastation/src/pages/Index/Index.tsx]: I see something I don't understand:

click to navigate to:
```
navigate(routeFor('/web/search'))}
```
I'm in:
image


click to navigate to:
```
navigate(routeFor('/plugin/massalabs/wallet/web-app/index'))
```
I'm in:
image

Comment (mazmassa) [reply web/massastation/src/pages/Index/Index.tsx]: > maybe not here, let's try

Ahh ok! Didn't know! My bad then!

Let's remove it!

Comment (lilyanB) [reply web/massastation/src/pages/Index/Index.tsx:123]: I remove it, done

**Pull request merged**